logging/bin/deploy_elasticsearch_open.sh

```diff
@@ -185,14 +214,12 @@ if [ ! -f "$TMP_DIR/$odfe_tgz_file" ]; then
    log_info "Cloning Open Distro for Elasticsearch repo"
    git clone https://github.com/opendistro-for-elasticsearch/opendistro-build
 
-   # checkout specific commit
    cd opendistro-build
-   git checkout 2c139044ee31a490b58ac6a306a5a5ef5ef21383
 
    # build package
    log_info "Packaging Helm Chart for Elasticsearch"
 
-   # cd opendistro-build/helm/opendistro-es/
+   # cd to the appropriate sub-directory
```

Comment:
Comment isn't very helpful..maybe just remove it.

Reply:
Will remove it.